### DISCUSSION TITLE
Measure growth of repos with Lang=solidity

### DISCUSSION THREAD

**Discussion by ckathleen** [community/community#24466] [Apps, API and Webhooks]
<p>I am trying to measure the month over month growth rate of the # of repos tagged with language:solidity. Can I do this through the github api? Any docs or help would be great.</p>

**Comment by lee-dohm** [comment, marked as answer]
<p>You can do this either manually by using the GitHub UI or you could write a script to access the <a href="https://developer.github.com/v3/search/#search-repositories" rel="nofollow noopener">GitHub Search API</a>. Specifically you would want to use the <a href="https://help.github.com/articles/searching-for-repositories/#search-by-language" rel="nofollow noopener"><code>language:</code> specifier</a> in your search.</p>
<p>I hope that helps!</p>
